src/decompiler.cc

```diff
@@ -826,9 +826,9 @@ struct Decompiler {
         }
       }
       if (is_import) {
-        s += ";";
+        s += cat("; // f", std::to_string(func_index));
```

Comment:
How about `index:` instead of `f`?

Reply:
I think there are other types that are also indexed? So a unique prefix seems better, can change to `func` if it makes more sense.

Reply:
Yes, I've just never seen `f0`, `f1` used to refer to function indexes, so its not obvious to me as the reader what it means.